hera_mc/mc_session.py

```diff
@@ -825,6 +825,72 @@ def get_weather_data(self, starttime, stoptime=None, variable=None):
                                  stoptime=stoptime, filter_column='variable',
                                  filter_value=variable)
 
+    def add_roach_temperature(self, time, roach, ambient_temp, inlet_temp,
+                              outlet_temp, fpga_temp, ppc_temp):
+        """
+        Add new roach temperature to the M&C database.
```

Comment:
For posterity, it might help to define what a "roach" is.

Reply:
I'll add a little more info.